samples/pg-sequence-alter.js

```diff
@@ -0,0 +1,94 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// sample-metadata:
+//  title: Alters a sequence in a PostgreSQL database table.
+//  usage: node pg-sequence-alter.js <INSTANCE_ID> <DATABASE_ID> <PROJECT_ID>
+
+'use strict';
+
+async function main(instanceId, databaseId, projectId) {
+  // [START spanner_postgresql_alter_sequence]
+  // Imports the Google Cloud client library.
+  const {Spanner} = require('@google-cloud/spanner');
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance';
+  // const databaseId = 'my-database';
+
+  // Creates a client
+  const spanner = new Spanner({
+    projectId: projectId,
+  });
+
+  async function alterSequence(instanceId, databaseId) {
+    // Gets a reference to a Cloud Spanner instance and database
+    const instance = spanner.instance(instanceId);
+    const database = instance.database(databaseId);
+
+    const request = ['ALTER SEQUENCE Seq SKIP RANGE 1000 5000000'];
+
+    try {
+      const [operation] = await database.updateSchema(request);
+
+      console.log('Waiting for operation to complete...');
+      await operation.promise();
+
+      console.log(
+        'Altered Seq sequence to skip an inclusive range between 1000 and 5000000.'
+      );
+    } catch (err) {
+      console.error('ERROR:', err);
+    }
+    database.runTransaction(async (err, transaction) => {
+      if (err) {
+        console.error(err);
+        return;
+      }
+      try {
+        const [rows, stats] = await transaction.run({
+          sql: "INSERT INTO Customers (CustomerName) VALUES ('Lea'), ('Catalina'), ('Smith') RETURNING CustomerId",
+        });
+
+        const rowCount = Math.floor(stats[stats.rowCount]);
+        console.log(
+          `Successfully inserted ${rowCount} record into the Customers table.`
+        );
+        rows.forEach(row => {
+          console.log(
+            `CustomerId: ${row.toJSON({wrapNumbers: true}).customerid.value}`
```

Comment:
Can we update the log statements (here and otherwise) in this PR based on client specification doc for consistency between clients?